app/src/main/java/com/keylesspalace/tusky/EditProfileActivity.kt

```diff
@@ -306,4 +320,17 @@ class EditProfileActivity : BaseActivity(), Injectable {
         Log.w("EditProfileActivity", "failed to pick media", throwable)
         Snackbar.make(binding.avatarButton, R.string.error_media_upload_sending, Snackbar.LENGTH_LONG).show()
     }
+
+    private fun showUnsavedChangesDialog() = lifecycleScope.launch {
+        when (launchAlertDialog()) {
+            AlertDialog.BUTTON_POSITIVE -> save()
+            else -> finish()
+        }
+    }
+
+    private suspend fun launchAlertDialog() = AlertDialog.Builder(this)
+        .setTitle(getString(R.string.title_edit_profile_save_changes_prompt))
```

Comment:
These dialogs don't need a title.

Reply:
I'll remove it, I still think they do need a title.